templates/single-tapestry.php

```diff
@@ -150,15 +151,59 @@ function enqueue_vue_app_build()
             var apiUrl = "<?php echo get_rest_url(null, 'tapestry-tool/v1'); ?>";
             var adminAjaxUrl = "<?php echo admin_url('admin-ajax.php'); ?>";
 
+            var globals = { recordAnalyticsEvent }
```

Comment:
I think moving forwards we should move global variables into this `globals` variable so it's more explicit where these variables are coming from.